transcom/mymove
# fix capturing the order type in adding orders detail info

## [I-13667](https://www13.v1host.com/USTRANSCOM38/Issue.mvc/Summary?oidToken=Issue%3A1046815)

## Summary

Orders Type wasn't being captured for the orders detail screen, need to capture and lock down when bluebark is selected and prevent the email from being sent
I
These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. run server with email backend configured to use aws >> `EMAIL_BACKEND=ses AWS_REGION=us-gov-west-1  aws-vault exec transcom-gov-dev -- make server_run`
2. Access the office app
3. Login as a services counlelor
4. create a new move with bluebark orders type
5. add your email as the personal email of the customer
6. on move details update orders - see that the orders type Is locked down to blueback
7. fill out rest of form, save
8. submit move/shiment
9. verify that no email was sent 

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
<img width="383" alt="image" src="https://github.com/user-attachments/assets/91090d57-ce5d-4ce2-a8ba-9285dcd8319d" />
